test/expected.txt

```diff
@@ -297,7 +297,14 @@
 || 
 || 0 persons listed!
 || ===================================================
-|| Enter command: || [Command entered:  exit]
+|| Enter command: || [Command entered:total]
+|| 1 Contacts found
+|| ===================================================
+|| Enter command: || [Command entered:topTen]
+||
+|| 0 persons listed!
```

Comment:
I don't get it, why 0 instead of 1?

Reply:
Also, seems like there should be a space between `Command entered:` and the command word, and also there should be no line break in between. Have you tried running the tests?